test/tests/pbs_smoketest.py

```diff
@@ -1318,7 +1319,9 @@ def test_resource_delete(self):
             self.server.status(RSC, id=self.resc_name)
             self.server.manager(MGR_CMD_DELETE, RSC,
                                 id=self.resc_name, logerr=False)
-        except:
+        except PbsManagerError:
+            pass
+        except PbsStatusError:
```

Comment:
There is a way to combine both of these to one except line.  I think its except (PbsManagerError, PbsStatusError):